src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/client/PpudAutomationApiClient.kt

```diff
@@ -42,6 +42,9 @@ class PpudAutomationApiClient(
   fun search(
     request: PpudSearchRequest,
   ): Mono<PpudSearchResponse> {
+    println("===========================================================================")
```

Comment:
Is this needed?

Reply:
? tks - surprised the linting didn't pick that up.